Project_source/app/src/main/java/android/clicker/school_live_simulator/TimerObserver.kt

```diff
@@ -1,6 +1,14 @@
 package android.clicker.school_live_simulator
 
+/**
+ * @author      Kirill Zheludev kirill.zheludev@yandex.ru
+ * @version     1.0                 (current version number of program)
+ * @since       1.0          (the version of the package this class was first added to)
+ */
+
 interface TimerObserver {
     fun timerTickHandler()
+	
     fun timerEndHandler()
```

Comment:
Where is documentation comments?

Reply:
Fixed, but I'm not sure about purposes of functions, so comments may be little incorrect